src/hooks/useMultiplayerState.ts

```diff
@@ -61,76 +61,76 @@ export function useMultiplayerState(roomId: string) {
     undoManager.redo();
   }, []);
 
-  /**
-   * Callback to update user's (self) presence
-   */
   const onChangePresence = useCallback((app: TldrawApp, user: TDUser) => {
-    if (!app.room) return;
-    room.setPresence({ id: app.room.userId, tdUser: user });
+    if (app.room) {
```

Comment:
Again, I would suggest getting back to the old version in which you don't have any additional (and unnecessary) nested code block.